### PR TITLE
Remove non specific code for windows/nix

### PR DESCRIPTION
**Pull request by achntrl**
### What does this PR do?

This PR removes windows specific code

### Motivation

Now that `openFile` doesn't block the file on Windows, we can reuse a lot of the Unix tailer code

### CODE REVIEW

File: pkg/logs/input/file/open_file_nix.go [hunk 1]
```diff
@@ -3,16 +3,15 @@
...
 
 import (
 	"os"
 )
 
-// DidRotate is not implemented on windows,
-// log rotations are handled by the tailer for now.
-func DidRotate(file *os.File, lastReadOffset int64) (bool, error) {
-	return false, nil
+// openFile open files with the standard Open method on *nix OSes
```
Comment (ajacquemot) [inline pkg/logs/input/file/open_file_nix.go]: nits: `openFile openS a file`

File: pkg/logs/input/file/open_file_windows.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// +build windows
+
+package file
+
+import (
+	"os"
+	"syscall"
+)
+
+// openFile reimplement the os.Open function for Windows because the default
```
Comment (ajacquemot) [inline pkg/logs/input/file/open_file_windows.go]: nits: `openFile reimplementS`

File: pkg/logs/input/file/tailer.go [hunk 3]
```diff
@@ -92,6 +93,57 @@ func (t *Tailer) Start(offset int64, whence int) error {
...
 }
 
+// setup sets up the file tailer
+func (t *Tailer) setup(offset int64, whence int) error {
+	fullpath, err := filepath.Abs(t.path)
+	if err != nil {
+		return err
+	}
+	t.tags = []string{fmt.Sprintf("filename:%s", filepath.Base(t.path))}
+	log.Info("Opening ", t.path)
```
Comment (ajacquemot) [inline pkg/logs/input/file/tailer.go]: nits: can we add a space between `l102` and `l103` please

File: pkg/logs/input/file/tailer.go [hunk 4]
```diff
@@ -92,6 +93,57 @@ func (t *Tailer) Start(offset int64, whence int) error {
...
+			// stop reading data from file
+			return
+		default:
+			// keep reading data from file
+			inBuf := make([]byte, 4096)
+			n, err := t.file.Read(inBuf)
+			if err != nil && err != io.EOF {
+				// an unexpected error occurred, stop the tailor
+				t.source.Status.Error(err)
+				log.Error("Err: ", err)
```
Comment (ajacquemot) [inline pkg/logs/input/file/tailer.go]: I think we could improve the message here

**Review by ajacquemot — COMMENTED** [review]
LGTM, thanks for this clean 👍

**Comment by tmichelet** [comment]
🎉

**Review by mnshdw — APPROVED** [review]

**Pull request merged**